scripts/bootstrap.js

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env babel-node
+
+import inquirer from 'inquirer';
+import program from 'commander';
+import childProcess from 'child_process';
+
+const spawn = childProcess.spawnSync;
+
+program
+  .version('1.0.0')
+  .option('--all', 'Run all without asking')
```

Comment:
Should `--all` be the default and instead a `--interactive` or `--pick` flag would launch the interactive picker?